include/macros.h

```diff
@@ -137,4 +142,17 @@ extern GraphicsContext* __gfxCtx;
 #define ASSERT(cond, msg, file, line) ((cond) ? ((void)0) : __assert(msg, file, line))
 #endif
 
+#define gSetTileCustom(pkt, fmt, siz, width, height, pal, cms, cmt, masks, maskt, shifts, shiftt)                      \
+    do {                                                                                                               \
+        gDPPipeSync(pkt);                                                                                              \
```

Comment:
Yeah it's basically a gbi macro so it's named as such. I moved it from `z_en_mag.h` to `macros.h` since it's also used in kaleido now. Maybe it should just be put in `gbi.h`, but I didn't do that because it's not part of any known gbi and it's only used in 2 places with specific arguments so I don't even know if the macro itself is correct. We should probably confirm it works for all possible uses first, and figure out its purpose.

Reply:
imo the prefix should be `gDP`, since all gbi commands are either prefixed `g*DP` or `g*SP`, and `gDP` makes most sense here.

Reply:
Alright, I changed it to `gDP`. I named it without DP or SP originally because I wasn't sure which one it should be, but DP makes sense.